fe/fe-core/src/main/java/org/apache/doris/deploy/DeployManager.java

```diff
@@ -148,38 +146,51 @@ public DeployManager(Env env, long intervalMs) {
     // Derived Class can override this method to init more private env variables,
     // but must class the parent's init at first.
     protected void initEnvVariables(String envElectableFeServiceGroup, String envObserverFeServiceGroup,
-            String envBackendServiceGroup, String envBrokerServiceGroup) {
+            String envBackendServiceGroup, String envBrokerServiceGroup, String envCnServiceGroup) {
 
         this.electableFeServiceGroup = Strings.nullToEmpty(System.getenv(envElectableFeServiceGroup));
         this.observerFeServiceGroup = Strings.nullToEmpty(System.getenv(envObserverFeServiceGroup));
         this.backendServiceGroup = Strings.nullToEmpty(System.getenv(envBackendServiceGroup));
         this.brokerServiceGroup = Strings.nullToEmpty(System.getenv(envBrokerServiceGroup));
+        this.cnServiceGroup = Strings.nullToEmpty(System.getenv(envCnServiceGroup));
 
-        LOG.info("get deploy env: {}, {}, {}, {}", envElectableFeServiceGroup, envObserverFeServiceGroup,
-                 envBackendServiceGroup, envBrokerServiceGroup);
+        LOG.info("get deploy env: {}, {}, {}, {}, {}", envElectableFeServiceGroup, envObserverFeServiceGroup,
+                envBackendServiceGroup, envBrokerServiceGroup, envCnServiceGroup);
 
-        // electableFeServiceGroup and backendServiceGroup must exist
-        if (Strings.isNullOrEmpty(electableFeServiceGroup) || Strings.isNullOrEmpty(backendServiceGroup)) {
-            LOG.warn("failed to init service group name."
-                    + " electableFeServiceGroup: {}, backendServiceGroup: {}",
-                     electableFeServiceGroup, backendServiceGroup);
-            System.exit(-1);
```

Comment:
I don't remember why we exit if service group does not exist.
But are you sure it is OK to remove it?

Reply:
This is to be compatible with only cn running on k8s